eng/packages/http-client-csharp-mgmt/emitter/src/emitter.ts

```diff
@@ -12,17 +12,16 @@ import {
 import { azureSDKContextOptions } from "./sdk-context-options.js";
 import { updateClients } from "./resource-detection.js";
 
-export async function $onEmit(context: EmitContext<AzureEmitterOptions>) {
+export async function $onEmit(context: EmitContext<AzureEmitterOptions>) {  
+
+  function updateCodeModel(codeModel: CodeModel, sdkContext: CSharpEmitterContext): CodeModel {
+    updateClients(codeModel, sdkContext);
+    return codeModel;
+  }
```

Comment:
could we move this function to the end of this function?
now it looks a bit confusing

Reply:
updated